src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
No. You have

List
..HeadingA
..Item1
..Item2
..HeadingB
..Item3
..Item4

The items are on the very same level as the headings. You do not have a nesting. You rather build that nesting at rendering time (if at all).

Where does the requirement that only one group is opened at a time suddenly come from? You could still manipulate the groups on the level of the listing, exactly like you could do in the divider case.

Where does the requirement to have multiple nesting of groups suddenly come from? Just like with the heading, you could have a level parameter on the group, pass it from the list, pass it down to other groups, etc. If we want to have several levels of nestings at all.

The interface for the standard listing would look like this:

```php
/**	 
 * @param string $title Title of the Listing
 * @param array<\ILIAS\UI\Component\Group|\ILIAS\UI\Component\Item\Item> $items List items
 * @return \ILIAS\UI\Component\Panel\Listing\Standard
 */
 public function standard($title, $items);
```
so almost no change here, just exchanged divider by group.